atlasaction/testdata/circleci/migrate-auto-rebase.txtar

```diff
@@ -40,10 +42,18 @@ stdout 'Migrations rebased successfully'
 
 # The git history now is correct with the new file
 exec git log --oneline
-stdout 'Rebase migrations in migrations'
+stdout 'migrations: rebase migration files'
 stdout 'commit the conflict files'
 stdout 'second commit'
 stdout 'first commit'
+
+# Ensure files are in the right order
+hashFile ./migrations/atlas.sum
+stdout '\[\d{14}_foo\.sql \d{14}_bar\.sql \d{14}_boo\.sql\]'
```

Comment:
We need ensure this order correctly, if not. It cause incident on the user database.